governance/default.go

```diff
@@ -1108,12 +1116,35 @@ func GetGovernanceItemsFromChainConfig(config *params.ChainConfig) GovernanceSet
 			params.CommitteeSize: istanbul.SubGroupSize,
 		}
 
-		for k, v := range istanbulMap {
-			if err := g.SetValue(k, v); err != nil {
-				writeFailLog(k, err)
-			}
+		appendToReturn(istanbulMap)
+	}
+
+	// magma params
+	if config.IsMagmaForkEnabled(common.Big0) &&
+		config.Governance.KIP71 != nil {
+		kip71 := config.Governance.KIP71
+		governanceMap := map[int]interface{}{
+			params.LowerBoundBaseFee:         kip71.LowerBoundBaseFee,
+			params.UpperBoundBaseFee:         kip71.UpperBoundBaseFee,
+			params.GasTarget:                 kip71.GasTarget,
+			params.MaxBlockGasUsedForBaseFee: kip71.MaxBlockGasUsedForBaseFee,
+			params.BaseFeeDenominator:        kip71.BaseFeeDenominator,
+		}
+
+		appendToReturn(governanceMap)
+	}
+
+	// kore params
+	if config.IsKoreForkEnabled(common.Big0) &&
+		config.Governance != nil {
+		governance := config.Governance
+		governanceMap := map[int]interface{}{
+			params.GovParamContract: governance.GovParamContract,
 		}
+
```

Comment:
```suggestion
```

Reply:
Do you mean deleting all LFs above before `appendToReturn`?

Reply:
there are 4 of them

Reply:
Yes, but it's okay to go over.